taxcalc/parameters.py

```diff
@@ -358,12 +355,8 @@ def _update(self, year_mods):
             # determine indexing status of parameter with name for year
             if name.endswith('_cpi'):
                 continue  # handle elsewhere in this method
-            if name in self._vals:
-                vals_indexed = self._vals[name].get('cpi_inflated', False)
-                integer_values = self._vals[name].get('integer_value')
-            else:
-                msg = 'parameter name {} not in parameter values dictionary'
-                raise ValueError(msg.format(name))
+            vals_indexed = self._vals[name].get('cpi_inflated', False)
+            integer_values = self._vals[name].get('integer_value')
             name_plus_cpi = name + '_cpi'
```

Comment:
Paths to this section of code:

`Policy.implement_reform` --> `ParametersBase._update`
`Behavior.update_behavior` --> `ParametersBase._update`

These are both called after calling the Policy and Behavior classes' respective `_validate_param_names_types` methods. Thus, we can be certain that the name is in `self._vals`. Also, I think an equivalent condition `pname not in self._vals` is already in `Policy._validate_param_names_types`[lines 446-450](https://github.com/open-source-economics/Tax-Calculator/blob/master/taxcalc/policy.py#L446-L450)  and was added to `Behavior._validate_param_names_types`.